react/Header/Header.less

```diff
@@ -162,6 +168,9 @@
 .locale {
   position: absolute;
   right: 0;
+  @media only screen and (min-width: @responsive-breakpoint) and (max-width: (@grid-container-width + @grid-gutter-width - 1)) {
```

Comment:
I thought about that but it's quite bespoke. It's a strange range between device and grid container width.